spark/v3.3/spark-extensions/src/main/antlr/org.apache.spark.sql.catalyst.parser.extensions/IcebergSqlExtensions.g4

```diff
@@ -73,14 +73,22 @@ statement
     | ALTER TABLE multipartIdentifier WRITE writeSpec                                       #setWriteDistributionAndOrdering
     | ALTER TABLE multipartIdentifier SET IDENTIFIER_KW FIELDS fieldList                    #setIdentifierFields
     | ALTER TABLE multipartIdentifier DROP IDENTIFIER_KW FIELDS fieldList                   #dropIdentifierFields
-    | ALTER TABLE multipartIdentifier createReplaceBranchClause   #createOrReplaceBranch
+    | ALTER TABLE multipartIdentifier createReplaceBranchClause                             #createOrReplaceBranch
+    | ALTER TABLE multipartIdentifier createTagClause                                       #createTag
+    ;
+
+createTagClause
+    : CREATE TAG (IF NOT EXISTS)? identifier tagOptions
```

Comment:
not sure if you finished all updates or not, but I think we can also add REPLACE part here and make it `#createorReplaceTag`?